src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -359,7 +359,10 @@ def __init__(
             irrelevant_patterns.update(exclude_paths_in_mount)
         self._fiter_paths = irrelevant_patterns
 
-    def snapshot(self) -> list[Table]:
+    def snapshot(self, *, force_refresh: bool = True) -> list[Table]:
+        # TODO: Figure out what this should do (and refactor); non-trivial due to it overwriting the static table data.
```

Comment:
yes!

Reply:
@JCZuurmond: Do you think the changes in a624f31 are sufficient for now?